torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -314,6 +314,331 @@ TensorView* TensorView::rFactor(const std::vector<int>& axes) {
   return producer;
 }
 
+// Create a TensorView before the original tensor. A common use case is to write
```

Comment:
no need to duplicate the function comments (for both declaration and definition) -> comments on the declaration (interface) only